### PR TITLE
Working version for gitlab integration

### PR DESCRIPTION
**Pull request by Scriptkiddi**

### CODE REVIEW

**Review by ghost — DISMISSED** [review]
no

File: meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  license: CC-BY-SA
+
+  min_ansible_version: 2.4
+
+  platforms:
+    - name: Ubuntu
+      versions:
+        - xenial
+        - yakkety
```
Comment (SlothOfAnarchy) [inline meta/main.yml]: yakkety and zesty are not supported anymore, but bionic should be

File: README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+| `mattermost_gitlab_integration_server_port`    | `5000`                                   | Port under which the server listens to webhooks from gitlab                                                      |
+| `mattermost_gitlab_integration_user`           | `www-data`                               | User under which the server should run. The user has to exist                                                    |
+| `mattermost_gitlab_integration_group`          | `www-data`                               | Group under which the server should run. The group hast to exist                                                 |
+| `mattermost_gitlab_integration_events`         | `[issue, comment, merge_request, build]` | List of events to be posted                                                                                      |
+| `mattermost_gitlab_integration_username`       | ` `                                      | Username to post under in Mattermost                                                                             |
```
Comment (SlothOfAnarchy) [inline README.md]: If there is no default, leave the column empty or use a cross emoji

File: README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+| `mattermost_gitlab_integration_icon_url`       | ` `                                      | URL to icon file which should show up in Mattermost                                                              |
+| `mattermost_gitlab_integration_channel`        | ` `                                      | Leave this blank to post to the default channel of your webhook                                                  |
+| `mattermost_gitlab_integration_verify_ssl`     | `true`                                   | Verify SSL certificates when POSTing to GitLab                                                                   |
+
+
+### Events
+
+Event                                                                                   | Enabled by default       | Variable name   |
+----------------------------------------------------------------------------------------|--------------------------|-----------------|
+[push](http://doc.gitlab.com/ee/web_hooks/web_hooks.html#comment-events)                | :heavy_multiplication_x: | `push`          |
```
Comment (SlothOfAnarchy) [inline README.md]: links to comment section instead of push

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

**Comment by Scriptkiddi** [comment]
@SlothOfAnarchy pls rerveiw

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**